test-crawler/collector/lang.go

```diff
@@ -0,0 +1,19 @@
+package collector
+
+type UnsupportedLanguageError struct{}
+
+func (m *UnsupportedLanguageError) Error() string {
+	return "Language not supported."
+}
+
+func getLangExt(lang_mode string) (string, error) {
+	switch lang_mode {
+	case "go":
+		return ".go", nil
+	case "rust":
+		return ".rs", nil
+	default:
+		// get all files regardless of extension
```

Comment:
Comment is wrong, this will return an error, not all files

Reply:
Right, i forgot to change that one, thanks